src/backend/.docker/Dockerfile

```diff
@@ -1,19 +1,13 @@
-FROM node:18.9-alpine
+FROM node:20.5-bullseye-slim
```

Comment:
is there a performance difference between alpine and bullseye-slim? I guess slim is more lightweight and seems to be based on ubuntu.

Reply:
Yes! Lightweight so less vulnerabilities, less space, better performance.